lib/response.js

```diff
@@ -958,9 +959,24 @@ res.render = function render(view, options, callback) {
   };
 
   // render
-  app.render(view, opts, done);
+  app.render(view, varStack, done);
 };
 
+/**
+ * Add `options` to the top of the reponse's template variables stack.
+ * Creates the stack the first time called.
+ *
+ * @public
+ */
+res.addTmplVars = function addTmplVars(options) {
```

Comment:
Yeah, no argument here.  I just wrote down the first thing that popped into my head.